testsuite/tests/lib-dynlink-csharp/Makefile

```diff
@@ -30,7 +30,9 @@ all: prepare bytecode bytecode-dll native native-dll
 prepare:
 	@if $(SUPPORTS_SHARED_LIBRARIES); then \
```

Comment:
Fine - though for consistency, can it be changed in the other four places too ("which" is just where I lifted it from!)?

Is this worth popping into 4.04? If @infinity0 doesn't have time today, I can cherry-pick the other part and do this later...